app/pages/Accounts/VerifyAddress.tsx

```diff
@@ -0,0 +1,133 @@
+            }
+            if (identity.version === 0) {
+                return 'The identity used to create this account uses a deprecated version of the key generation. It is recommended to create a new identity and use accounts on that instead.';
+            }
+            return undefined;
+        },
+        () => {},
+        [identity, account.address]
+    );
+
+    const ledgerCall = useCallback(
+        async (ledger: ConcordiumLedgerClient) => {
+            setOpened(true);
+            if (credential === undefined) {
+                setWarning(
+                    'No credentials, that were created by the current Ledger, were found on this account. Please verify that the connected Ledger is for this account.'
+                );
+            } else if (credential.credentialIndex !== 0) {
+                setWarning(
+                    "An account's address is derived from the initial credential on that account. The initial credential on this account was not created from the currently connected Ledger, and therefore it is not possible to verify the address of the account."
+                );
+            } else {
+                await ledger.verifyAddress(
+                    credential.identityNumber,
+                    credential.credentialNumber
+                );
+            }
+            setOpened(false);
+        },
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+        [credential?.identityNumber, credential?.credentialNumber]
+    );
+
+    return (
+        <>
+            <MessageModal
+                title="Unable to verify account address."
+                message={warning}
+                buttonText="Okay"
+                onClose={() => setWarning(undefined)}
+                open={Boolean(warning)}
+                disableClose
+            />
+            <Ledger ledgerCallback={ledgerCall}>
+                {({ isReady, statusView, submitHandler = asyncNoOp }) => (
+                    <>
+                        {reasonForNotBeingVerifiable && (
+                            <>
+                                {display}
+                                <p className="textCenter">
+                                    {reasonForNotBeingVerifiable}
+                                </p>
+                            </>
+                        )}
+                        {!reasonForNotBeingVerifiable && (
+                            <>
+                                {opened || (
+                                    <div
+                                        className={clsx(
+                                            className,
+                                            'flexColumn'
+                                        )}
+                                    >
+                                        <h3 className="mB40 mT0">
+                                            Unlock Ledger to verify and show
```

Comment:
This text is not centered. Also I am not sure about the text. It asks me to unlock even though my device is unlocked and connected. Perhaps we don't even need the text? Perhaps Jens can help us here...

Reply:
Removed it.